package.json

```diff
@@ -1,225 +1,219 @@
 {
-  "name": "Buttercup",
```

Comment:
Please revert this formatting - including indentation and reordering. I'd like it to show only the modifications that you've needed to make.